### DISCUSSION TITLE
Welcome to GithubClient Discussions!

### DISCUSSION THREAD

**Discussion by ExistOrLive** [MengAndJie/GithubClient#19] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ExistOrLive** [comment]
Hello

**Reply by ExistOrLive** [reply]
测试

**Reply by ExistOrLive** [reply]
测试

**Reply by ExistOrLive1** [reply]
> 测试

测试

**Comment by scrutinizer11** [comment]
Hello,

Thanks for your friendly app, especially in the wake of other devs pulling their apps from the App Store or GitHub raising the incompatibility bar leaving me outboard. Could you make it use the app view instead of the web view where applicable (currently, the GitHub discussions are viewable only through the web view)?

iPhone SE, iOS 12.4.1

**Reply by ExistOrLive** [reply]
Thanks for your advice，we will arrange the task as soon as possible

**Comment by ExistOrLive** [comment]
测试
